Tests/FrameProviderTests.swift

```diff
@@ -171,7 +171,7 @@ final class FrameProviderTests: XCTestCase {
         itemType: .dayOfWeekInMonth(
           position: .fourth,
           month: Month(era: 1, year: 2020, month: 01, isInGregorianCalendar: true)),
-        frame: CGRect(x: 130, y: 200, width: 40, height: 40))
```

Comment:
the made up 40x40 size used to work because the calculation in FrameProvider was based on `frame.minY`. Now that the calculation is based on `frame.maxY`, we can't just use fake size values. Instead, this should be the real day size from `frameProvider.daySize`, otherwise the resulting calculations will be nonsense.